test/MeshBVH.options.test.js

```diff
@@ -273,4 +273,68 @@ describe( 'Options', () => {
 
 	} );
 
+	describe( 'range', () => {
+
+		let geometry;
+		beforeEach( () => {
+
+			geometry = new TorusGeometry( 5, 5, 400, 100 );
+
+		} );
+
+		it( 'should respect the range option without groups.', () => {
```

Comment:
I think we'll want to make sure this works with the `indirect` argument, as well. Do you mind adding another test or two for that case?